.test-infra/jenkins/common_job_properties.groovy

```diff
@@ -236,10 +218,19 @@ class common_job_properties {
     }
   }
 
+  static def mapToArgList(LinkedHashMap<String, String> inputArgs) {
```

Comment:
<!--new_thread; commit:13434ff7764e80f3ff96f6120b9100f537df0f36; resolved:0-->
This returns a single String, not a list as the name suggests (right?)